### DISCUSSION TITLE
Wemos D1 r2 + EthernetENC

### DISCUSSION THREAD

**Discussion by dBarmpxkos** [Networking-for-Arduino/EthernetENC#8] [Q&A] [answered]
Hello,

I am trying to use the library with a Wemos D1 r2, I can see from the docs it has been tested and works. Could you share any information on the connections you used to test? Any buffer, resistor between the lines? 

Thanks and keep up the good work

**Comment by JAndrassy** [comment, marked as answer]
use direct connection of SPI pins and a CS pin different then io 15. enc28j60 is a 3.3 V device.

**Reply by dBarmpxkos** [reply]
Works out of the box! Many thanks
